### DISCUSSION TITLE
Linbox For 86Box

### DISCUSSION THREAD

**Discussion by Dungeonseeker** [86Box/86Box#2324] [Show and tell]
I made an incredibly simple frontend for 86Box Linux in Gambas. ATM it requires you to install Gambas and compile it yourself, I'll build some standalone packages once its been field tested.

Anybody who knows visual basic, I've released it under Apache 2 and I'm actively encouraging anybody to fork it and build on it. I genuinely do not care what anybody does with it as long as I'm credited somewhere.

I also borrowed a few icons from Winbox, not sure if this affects the license at all?

Please break it, I'm a new coder and this is my first serious attempt to create something so its almost certain I did something wrong :) 

I'll also try to actively accept PRs.

https://github.com/Dungeonseeker/linbox-qt5

**Comment by ts-korhonen** [comment]
Nice! Good to see developers take interest in 86Box.

> I also borrowed a few icons from Winbox, not sure if this affects the license at all?

Since Winbox is GPL-3.0 you need to retain the license. I think this also means that the combination of the icons and your manager is automatically GPL-3.0, but if the icons are left out (e.g. when someone forks it or you get new icons) it can be Apache 2 again. See https://www.apache.org/licenses/GPL-compatibility.html

**Comment by jriwanek** [comment]
Edit: Corrected link to it's current location.
